File: codingbat/python/src/logic-2/make_chocolate.py
def make_chocolate(small, big, goal):
  big_used = (goal-(goal%5))/5 if (goal-(goal%5))/5 <= big else big;
  if goal-(big_used*5) <= small or goal - (big_used * 5) - small <= 0:
    return goal - big_used*5;
  return -1;

File: codingbat/java/src/logic-2/makeChocolate.java
public int makeChocolate(int small, int big, int goal) {
  int bigUsed = (int)goal/5;
  if(small >= goal-(5*(bigUsed > big ? big : bigUsed))){
    return goal-(5*(bigUsed > big ? big : bigUsed));
  }
  return -1;
}
